Add documentation for Hubspot OAuth

edmundito opened the pull request:
## Change description

This change updates the docs with documentation about connecting to Hubspot with Auth.

## Checklists

### Development

- [x] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

edmundito committed:
Add documentation for Hubspot OAuth

edmundito commented on components/ImageInBrowserFrame.tsx:
```diff
@@ -20,7 +20,7 @@ export default function ImageInBrowserFrame({
     <div
       style={{
         padding: "20px 0 0",
-        margin: 10,
```
This change ensures that there's enough margin for the drop shadow around the screenshot pages so that we don't need to add 2 `<br>` after them. Even though I fixed it in just the Hubspot doc, after we merge, we should clean up the other pages.

Adorism replied:
Oh good. Yes, I didn't like it if the text landed in the shadow. I know the integration docs have lots of double <br> going on and I can clean those up.

Adorism commented on pages/docs/integrations/grouparoo-hubspot.mdx:
```diff
@@ -87,7 +76,31 @@ In Grouparoo, you specify the options to connect to HubSpot and then test the co
+  height={526}
+/>
+
+If the token becomes invalid, you can always return to the app page to re-request a new token.
```
```suggestion
If the token becomes invalid, you can revisit the App page to request a new one.
```

Adorism reviewed (approved):
OAuth is exciting! This looks great.

krishnaglick reviewed (approved):
I think there's a typo on line 74: `hapikey`

edmundito committed:
Remove hapikey reference and update Hubspot to HubSpot

edmundito commented:
@krishnaglick that's the name of the value int he actual API but it makes sense not to mention it here because you'd only know this if you've used the hubspot API before. I removed this and also fixed the casing in the name HubSpot.

krishnaglick commented:
> @krishnaglick that's the name of the value int he actual API but it makes sense not to mention it here because you'd only know this if you've used the hubspot API before. I removed this and also fixed the casing in the name HubSpot.

Good to know! I wasn't sure, but I spotted it in the docs so wanted to make sure :)
